src/tools/tidy/src/lib.rs

```diff
@@ -33,10 +33,9 @@ macro_rules! t {
 
 macro_rules! tidy_error {
     ($bad:expr, $fmt:expr, $($arg:tt)*) => ({
-        use std::io::Write;
         *$bad = true;
-        write!(::std::io::stderr(), "tidy error: ").expect("could not write to stderr");
-        writeln!(::std::io::stderr(), $fmt, $($arg)*).expect("could not write to stderr");
+        eprint!("some tidy checks failed");
```

Comment:
Did you intent to change this from `"tidy error: "` to `"some tidy checks failed"`?

Reply:
Nope, good catch! Fixed in the latest force push